pybikes/metro_bike_share.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+class MetroBikeShare(BikeShareSystem):
+
+    sync = True
+
+    meta = {
+        'system': 'Metro Bike Share',
+        'company': ['Los Angeles County Metropolitan Transportation Authority (Metro)', 'Bicycle Transit Systems Inc', 'B-cycle'],
+    }
+
+    def __init__(self, tag, feed_url, meta):
+        super(MetroBikeShare, self).__init__(tag, meta)
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+            scraper.ssl_verification = False
```

Comment:
> Since we are not checking on pybikes installation the python version, maybe it's a good idea to keep the verification off, with a comment on why it's like that

First sorry for the late answer. Guess you are right to leave a comment in the code as you don't pin the python version in this project. I'm going to add this like you already commented.